app/Resources/assets/js/lsdoc/index.js

```diff
@@ -679,7 +679,10 @@ $(document).ready(adjustWindow);
 $(window).on('resize', adjustWindow);
 
 $(document).ready(function () {
-    let table = $('#datatable').DataTable();
+    let table = $('#datatable').DataTable({
+        order: [],
+        columnDefs: [ {orderable: false, targets: [0]}, {orderable: false, targets: [4]} ]
```

Comment:
This may be why the automated tests are no longer passing, as it seems the column used to sort the table to get the last user added is no longer sortable.